From 1fc30ef8f7f7376a3191de5d25fe41aa56f7db53 Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Mon, 28 Mar 2022 19:33:22 -0700
Subject: [PATCH] udev rules: add kvm rule

Based on commit e8c942d0df396e3c756a7261a8bffcb8c9847a1a.

Make /dev/kvm available to members of the kvm group. This improves
security since it avoids having to be root to make use of kvm.

Signed-off-by: Vu Tran <vu.tran@windriver.com>
Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>

Update context of the patch on 20210727.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Update context of the patch on 20220329

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>

Update context of the patch on 20220614.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 rules.d/99-kvm.rules | 1 +
 rules.d/meson.build  | 1 +
 2 files changed, 1 insertions(+), 0 deletion(-)
 create mode 100644 rules.d/99-kvm.rules

diff --git a/rules.d/99-kvm.rules b/rules.d/99-kvm.rules
new file mode 100644
index 0000000000..c2f7317aac
--- /dev/null
+++ b/rules.d/99-kvm.rules
@@ -0,0 +1 @@
+KERNEL=="kvm", GROUP="kvm", MODE="0660"
diff --git a/rules.d/meson.build b/rules.d/meson.build
index 8d2878a36d..71d9939a5b 100644
--- a/rules.d/meson.build
+++ b/rules.d/meson.build
@@ -28,6 +28,7 @@ rules = [
                '78-sound-card.rules',
                '80-net-setup-link.rules',
                '81-net-dhcp.rules',
+               '99-kvm.rules',
               )],
 
         [files('80-drivers.rules'),
-- 
2.33.0

